integration-tests/bats/sql-local-remote.bats

```diff
@@ -108,3 +108,29 @@ teardown() {
     [[ "$output" =~ "defaultDB does not exist" ]] || false
 }
 
+@test "sql-local-remote: verify dolt blame behavior is identical in switch between server/no server" {
+    cd defaultDB
```

Comment:
There is a bug here where we are out of sync with other command behavior. You shouldn't need to `cd defaultDB` - and in fact I tested this and you get an error stating that you need to dolt init first. This is incorrect behavior.

Reply:
I'm not sure I understand why I don't need to do `cd defaultDB` first. I would need to be in a dolt repo to do `dolt add` and `dolt commit` right?